### PR TITLE
[MM-28965] - Prevent link overlap in channel header

### PR DESCRIPTION
**Pull request by nevyangelova**
#### Summary
Quick fix for links that overlap in channel header

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28965

Fix v5.29

### CODE REVIEW

File: components/markdown_image.jsx [hunk 1]
```diff
@@ -113,9 +113,7 @@ export default class MarkdownImage extends React.PureComponent {
                                 rel='noopener noreferrer'
                                 target='_blank'
                                 title={this.props.title}
-                            >
-                                {alt}
-                            </a>
+                            />
```
Comment (hmhealey) [inline components/markdown_image.jsx:116]: I don't know if this needs to be changed. If someone links to an unsafe image, it won't show anything now whereas it made a link to that image before.

File: sass/layout/_headers.scss [hunk 2]
```diff
@@ -269,6 +269,7 @@
             .emoticon {
                 position: relative;
                 top: -1px;
+                overflow: hidden;
```
Comment (hmhealey) [inline sass/layout/_headers.scss:272]: I didn't know the header had its own styling for emojis. I'm not at all surprised this broke then

**Review by hmhealey — CHANGES_REQUESTED** [review]

File: components/markdown_image.jsx (hunk 1 above)
Comment (nevyangelova) [reply components/markdown_image.jsx:116]: Oh I see, thanks for clarifying

**Review by hmhealey — APPROVED** [review]

**Review by calebroseland — APPROVED** [review]

**Comment by amyblais** [comment]
@nevyangelova From your investigation do you know if this is a regression in v5.27?

**Comment by nevyangelova** [comment]
@amyblais it isn't these styles were implemented a year ago.

**Comment by ThiefMaster** [comment]
I'm relatively sure that this problem only started showing up during one of the last two releases. I had the header that's mentioned in the JIRA ticket for a long time and clicking the link there worked fine.

**Comment by nevyangelova** [comment]
@ThiefMaster it's possible that some styles were removed and revealed this bug but this is the proper fix and it wasn't implemented correctly a year ago.

**Comment by ThiefMaster** [comment]
I just wanted to point out that it'd probably be a good candidate for a bugfix release if there's going to be one since it's a small but annoying bug.

**Comment by nevyangelova** [comment]
@ThiefMaster makes sense, @amyblais should this go in 5.28?

**Comment by amyblais** [comment]
I think v5.29 is ok.

**Review by jgilliam17 — APPROVED** [review]
Thank you @nevyangelova 
Tested, looks good to merge.
- Verified header description links are clickable and they are not overlapping.
Note: The line that appears on mouse hover over linked emoji can't be removed currently and will have to be addressed separately if we decide to change/remove how markdown is processed.

**Pull request merged**